sunbird_service_api_test/src/test/java/org/sunbird/integration/test/malformedRequest/MalformedRequestTest.java

```diff
@@ -1,123 +1,132 @@
-						"/api/user/v1/create" },
-				new Object[] { emptyPayLoad,
-						Constant.MALFORMED_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createOrg",
-						"/api/org/v1/create" },
-				new Object[] { emptyPayLoad,
-						Constant.MALFORMED_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createCourse",
-						"/api/course/v1/batch/create" },
-				new Object[] { emptyPayLoad,
-						Constant.MALFORMED_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createPage",
-						"/api/data/v1/page/create" },
-				new Object[] { emptyPayLoad,
-						Constant.MALFORMED_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createNote",
-						"/api/notes/v1/create" } };
-	}
-
-	/**
-	 * Test for create request without content-type header.
-	 *
-	 * @param requestJson - request input json
-	 * @param responseJson - response output json
-	 * @param testName - name of the name
-	 * @param url - url of API
-	 */
-	@Test(dataProvider = "createRequestDataProvider", dependsOnMethods = { "getAdminAuthToken" })
-	@CitrusParameters({ "requestJson", "responseJson", "testName", "url" })
-	@CitrusTest
-	public void testRequestWithoutContentType(String requestJson, String responseJson, String testName, String url) {
-		getTestCase().setName(testName);
-		http().client(restTestClient).send().post(url)
-				.header(Constant.AUTHORIZATION, Constant.BEARER + initGlobalValues.getApiKey())
-				.header(Constant.X_AUTHENTICATED_USER_TOKEN, admin_token).payload(requestJson);
-
-		http().client(restTestClient).receive().response(HttpStatus.BAD_REQUEST)
-				.payload(new ClassPathResource(responseJson));
-
-	}
+  @Autowired private HttpClient restTestClient;
+  @Autowired private TestGlobalProperty initGlobalValues;
 
-	/**
-	 * Test for create request with invalid(example json-ld) content-type header.
-	 *
-	 * @param requestJson - request input json
-	 * @param responseJson - response output json
-	 * @param testName - name of the name
-	 * @param url - url of API
-	 */
-	@Test(dataProvider = "createRequestDataProvider", dependsOnMethods = { "getAdminAuthToken" })
-	@CitrusParameters({ "requestJson", "responseJson", "testName", "url" })
-	@CitrusTest
-	public void testRequestWithInvalidContentType(String requestJson, String responseJson, String testName, String url) {
-		getTestCase().setName(testName);
-		http().client(restTestClient).send().post(url).contentType(Constant.CONTENT_TYPE_APPLICATION_JSON_LD)
-				.header(Constant.AUTHORIZATION, Constant.BEARER + initGlobalValues.getApiKey())
-				.header(Constant.X_AUTHENTICATED_USER_TOKEN, admin_token).payload(requestJson);
+  @DataProvider(name = "createRequestDataProvider")
+  public Object[][] createRequestDataProvider() {
+    String emptyPayLoad = "{\"request\":{}}";
+    return new Object[][] {
+      new Object[] {
+        emptyPayLoad,
+        Constant.MALFORMED_TEMPLATE_LOCATION + "content_type_mandatory_response.json",
```

Comment:
Place request.json and response.json (and any other assets) in a separate folder for each test.

e.g. testMalformedCreateUser, testMalformedCreateOrg, testMalformedCreateBatch, testMalformedCreatePage, testMalformedCreateNote.